recipes/calysto_bash/meta.yaml

```diff
@@ -20,6 +20,8 @@ requirements:
     - pip
     - python
     - ipykernel
+  build:
+    - metakernel
```

Comment:
Shouldn't this be a host dependency?

Reply:
Sorry, I do not really know the difference between `host` and `build`. I see that  `python -m calysto_bash install` failed due to lack of `metakernel` so I added it, assuming it is required for installation (build).

Reply:
i think it is mostly compilers go in build. Not too clear on that either.

Reply:
Are these even needed in host.  They seem to be runtime deps.  host deps are those that are required for install on the target machine architecture

Reply:
The problem here is that the `install` step (`python -m calysto_bash install`) is handled by `metakernel`.